src/main/java/codeu/model/data/User.java

```diff
@@ -58,4 +65,32 @@ public String getPasswordHash() {
   public Instant getCreationTime() {
     return creation;
   }
+
+  /** Returns the number of messages sent by this User. */
+  public long getNumMessages() {
+    return numMessages;
+  }
+
+  /** Returns the number of messages sent by this User. */
+  public long getIncNumMessages() {
+    this.numMessages = numMessages + 1; 
```

Comment:
Right now both getNumMessages() and getIncNumMessages() "return the number of messages sent by this user". I think it might make sense to have each method have a specific task that it tries to do and then have it only do that task. Perhaps this method could be named incrementNumMessages() and we could change this to just: 
`
incrementNumMessages() {
  this.numMessages++;
}
`
So getNumMessages() only returns the this.numMessages and incrementNumMessages() only increments this.numMessages by one. If you wanted to increment the number of messages and also get the new number of messages, you could call incrementNumMessages() to increment and get the new number by calling getNumMessages().

Reply:
got it 👍